src/app/action-items/action-items.component.css

```diff
@@ -0,0 +1,113 @@
+/* flex datatable */
+
+* {
+    font-size: 16px;
+}
+
+:root {
+    --table-header: rgb(10, 44, 51);
+    --table-header-border: rgb(10, 44, 51);
+    --table-border: #d9d9d9;
+    --row-bg: #f4f2f1;
+}
+
+div {
+    box-sizing: border-box;
+}
+
+.table-container {
+    display: block;
+    margin: 2em auto;
+    width: 80%;
+    /* max-width: 600px; */
+    box-shadow: 0 6px 6px 0 rgb(148, 143, 143), 0 6px 15px 0 rgb(148, 143, 143);
+}
+
+.flag-icon {
+    margin-right: 0.1em;
+}
+
+.flex-table {
+    display: flex;
+    flex-flow: row wrap;
+    border-left: solid 1px var(--table-border);
+    transition: 0.5s;
+}
+
+.flex-table:first-of-type {
+    border-left: solid 1px var(--table-header-border);
+}
+
+.flex-table:first-of-type .flex-row {
+    background: var(--table-header);
+    color: rgb(104, 100, 100);
+    border-color: var(--table-header-border);
+}
+
+.flex-table.row:nth-child(odd) .flex-row {
+    background: var(--row-bg);
+}
+
+.flex-table:hover {
+    background: #f5f5f5;
+    transition: 500ms;
+}
+
+.flex-row {
+    width: calc(100% / 5);
+    text-align: center;
+    padding: 10px;
+    /* padding-top: 10px;
+    padding-bottom: 10px; */
+    border-right: solid 1px var(--table-border);
+    border-bottom: solid 1px var(--table-border);
+}
+
+.rowspan {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: flex-start;
+    justify-content: center;
+}
+
+.column {
+    display: flex;
+    flex-flow: column wrap;
+    width: 75%;
+    padding: 0;
+}
+
+.column.flex-row {
```

Comment:
You can do `sass nesting` - see more: https://dev.to/sarah_chima/nesting-in-sass-bme

Reply:
SASS nesting implemented.